src/lando/main/models/profile.py

```diff
@@ -1,15 +1,44 @@
 from __future__ import annotations
 
-from django.contrib.auth.models import User
+from django.contrib.auth.models import Permission, User
+from django.contrib.contenttypes.models import ContentType
 from django.db import models
 
 from lando.main.models.base import BaseModel
 
+SCM_PERMISSIONS = (
+    ("scm_conduit", "SCM_CONDUIT"),
+    ("scm_level_1", "SCM_LEVEL_1"),
+    ("scm_level_2", "SCM_LEVEL_2"),
+    ("scm_level_3", "SCM_LEVEL_3"),
+    ("scm_versioncontrol", "SCM_VERSIONCONTROL"),
+)
+
```

Comment:
This list is missing quite a few scm levels we support in Lando currently (`scm_nss`, `scm_firefoxci`, etc). See `repos.py` in LandoAPI. We should add all the SCM levels for repos we support in Lando.

Reply:
Makes sense, will add.